modules/ROOT/pages/VPNConfiguration.adoc

```diff
@@ -33,7 +34,7 @@ To request a VPN connection with Bonita Cloud, create a new case on the Customer
 * VPN gateway type
 * Gateway public IP address
 * Encryption algorithms (AES-128, AES-256)
-* Integrity Algorithms (SHA1, SHA2-256)
-* Authentication modes (pre-shared key/digital signature)
-* Diffie-Hellman group (2, 5, 14...)
+* Integrity Algorithms (SHA2-256, SHA2-384, SHA2-512)
+* Authentication modes (IKEv1, IKEv2 - used for pre-shared key/digital signature)
+* Diffie-Hellman group (2, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24)
```

Comment:
As you removed 5, we may also remove 2
```suggestion
* Diffie-Hellman group (14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24)
```

Reply:
5 isn't available while configuring "Phase 1 DH group numbers" AWS VPN, and 2 is available in phase 1 and 2.